controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -552,20 +545,22 @@ func (r *StorageConsumerReconciler) reconcileCephClientHealthChecker() error {
 		return nil
 	})
 
-	cephClientHealthCheckerDetails := &ocsv1alpha1.CephResourcesSpec{
-		Kind: r.cephClientHealthChecker.Kind,
-		Name: r.cephClientHealthChecker.Name,
+	if err != nil {
+		r.Log.Error(
+			err,
+			"Failed to update CephClient.",
+			"CephClient",
+			klog.KRef(r.cephClientHealthChecker.Namespace, r.cephClientHealthChecker.Name),
+		)
+		return err
 	}
+
+	var phase string
```

Comment:
To make it more obvious that we are willing to intentionally use an empty string
```suggestion
phase := "" 
```